packages/compose/package.json

```diff
@@ -21,10 +21,11 @@
 	"module": "build-module/index.js",
 	"dependencies": {
 		"@wordpress/element": "file:../element",
-		"@wordpress/is-shallow-equal": "file:packages/is-shallow-equal",
+		"@wordpress/is-shallow-equal": "file:../is-shallow-equal",
 		"lodash": "^4.17.10"
 	},
 	"devDependencies": {
+		"@wordpress/jest-console": "file:packages/jest-console",
```

Comment:
I don't think it should be included here. We never import it explicitly in the test file.